lms/djangoapps/ora_staff_grader/serializers.py

```diff
@@ -157,10 +159,55 @@ class Meta:
             "gradeStatus",
             "lockStatus",
             "score",
+            "email",
+            "fullname",
         ]
         read_only_fields = fields
 
 
+class AssessmentScoresSerializer(serializers.Serializer):
+    """
+    Assessment Score for displaying assessment table in Enhanced Staff Grader (ESG)
+    """
```

Comment:
We could change this to:
```suggestion
    """
    Score information associated with a specific assessment.
    
     This serializer was intended for displaying assessment information in the Enhanced Staff Grader (ESG).
    """
```